app/scripts/views/mixins/open-confirmation-email-mixin.js

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+/**
+ * A view mixin that helps confirm* views show the "Open Gmail" button
+ * directly from FxA.
+ */
+define(function (require, exports, module) {
```

Comment:
nit: could we rename this from `open-confirmation-email-mixin.js` to `open-webmail-mixin.js`. any thoughts against this? does the new name make it easier to find and understand this mixin?